.github/workflows/tests.yml

```diff
@@ -0,0 +1,42 @@
+name: Tests
+
+on:
+    push:
+    pull_request:
+
+jobs:
+
+    test:
+        name: PHPUnit Tests
+        runs-on: ubuntu-latest
+
+        env:
+            COMPOSER_ROOT_VERSION: dev-main
+
+        strategy:
+            matrix:
+                php:
+                    - '7.1'
+                    - '7.2'
+                    - '7.3'
+                    - '7.4'
+                    - '8.0'
+            fail-fast: false
```

Comment:
this is opposite to previous conf in travis

Reply:
The `fast_finish` setting on Travis works slightly different than `fail-fast`, IIRC.

If I did not disable `fail-fast`, GH actions would cancel the whole matrix immediately if one of the matrix builds fail. That way, we would never know if we deal with a general failure or a version-specific one.

Right now this would mean: The php 8 build fails and the others are cancelled.